tests/unit/lib/hotkeyManager.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { it } from "vitest";
+import { HotkeyManager, HotkeysJs } from "@/plugins/hotkeyPlugin";
+
+type DummyHotkeysJs = HotkeysJs & {
+  registeredHotkeys: {
+    key: string;
+    scope: string;
+    // callbackを持たせると比較が面倒になるので持たせない
+    // callback: (e: KeyboardEvent) => void;
+  }[];
+  currentScope: string;
+};
+const createHotkeyManager = (): {
+  hotkeyManager: HotkeyManager;
+  dummyHotkeysJs: DummyHotkeysJs;
+} => {
+  const registeredHotkeys: DummyHotkeysJs["registeredHotkeys"] = [];
+  const dummyHotkeysJs = ((
+    key: string,
+    { scope }: { scope: string },
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    callback: (e: KeyboardEvent) => void
+  ) => {
+    registeredHotkeys.push({ key, scope });
```

Comment:
keyが登録済みだったらエラーにすべきかも
```suggestion
    if (registeredHotkeys.some((h) => h.key === key && h.scope === scope)) {
      throw new Error(`key: ${key} is already registered`);
    }
    registeredHotkeys.push({ key, scope });
```